pkg/assembler/helper.go

```diff
@@ -15,8 +15,19 @@
 
 package assembler
 
-import "reflect"
+import (
+	"reflect"
+	"strings"
+)
 
 func isDefined(v interface{}) bool {
 	return !reflect.ValueOf(v).IsZero()
 }
+
+func toLower(v []string) []string {
```

Comment:
how about using varargs for this? will that work well?

Reply:
changed to variadic function but it would still need to return a `[]string`